.env_example

```diff
@@ -3,4 +3,16 @@ GO_API_PORT=3000
 AE_NODE_URL="http://localhost:3001"
 SECRET_KEY="xxxxxxxxxxxxx" 
 CONTRACT_SOURCE_URL_GIT_RAW="https://raw.githubusercontent.com/LimeChain/aepp-token-migration-smart-contract/master/contracts/TokenMigration.aes" 
-AE_CONTRACT_TOKEN_MIGRATION_ADDRESS="ct_nvYGScRcwHWurpUWg9qPyWNQKxnSxG55zXoWSQigMSYSa8g51"
\ No newline at end of file
+AE_CONTRACT_TOKEN_MIGRATION_ADDRESS="ct_nvYGScRcwHWurpUWg9qPyWNQKxnSxG55zXoWSQigMSYSa8g51"
+
+AE_NETWORK_ID="ae_devnet"
+AE_BACKEND="fate"
+AE_ABI_VERSION=3
+
+BACKENDLESS_LOGIN="xxxx@xxxx.xxx"
```

Comment:
Are the Backendless variables needed for the deployment in GCloud? Would someone need to have these if deploying on another cloud provider?

Reply:
yes, variables needed or should refactor the code to work without backendless server. if variables exists, notify service or do nothing

Reply:
Lets make it as you suggested. If there is no backendless env variables we shall not send statistics to the backendless.

Reply:
done.